docs/sources/setup-grafana/installation/docker/index.md

```diff
@@ -99,7 +95,7 @@ To use persistent storage, complete the following steps:
 
 #### Use bind mounts
 
-If you plan to use folders on your host for the database or configuration when running Grafana in Docker, you must start the container with a user with permission to access and write to the folder you map.
+If you plan to use directories on your host for the database or configuration when running Grafana in Docker, you must start the container with a user with permission to access and write to the folder you map.
```

Comment:
Let's make sure we're consistent throughout the doc
```suggestion
If you plan to use directories on your host for the database or configuration when running Grafana in Docker, you must start the container with a user with permission to access and write to the directory you map.
```

Reply:
I did a search for all mentions of `folder` and replaced them with the term `directory`